TitanRobotics2024/src/main/java/frc/robot/Subsystem/SmartDashboardSubsystem.java

```diff
@@ -0,0 +1,90 @@
+package frc.robot.Subsystem;
+
+import java.util.ArrayList;
+import java.util.List;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class SmartDashboardSubsystem implements Subsystem
+{
+    private static SmartDashboardSubsystem instance = null;
+
+    private Gyro gyro;
```

Comment:
Is there even a Gyro on the robot?

Reply:
there REALLY should be if there's not already